docs/index.html

```diff
@@ -1,12 +1,17 @@
 <!doctype html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <title>Messaging Segment Calculator</title>
-  <link rel="stylesheet" href="./main.css">
-  <script type="text/javascript" src="./grapheme-splitter.js"></script>
-  <script type="text/javascript" src="./segments_calculator.js"></script>
-  <script type="text/javascript" src="./segments_viewer.js"></script>
+  <meta charset="utf-8" />
+    <title>Messaging Segment Calculator</title>
+    <link rel="stylesheet" href="styles/main.css" />
+    <script
+      src="https://cdn.jsdelivr.net/npm/grapheme-splitter@1.0.4/index.js"
+      integrity="sha256-ApHozg+22P2OAAiCbUVy12oWNZxACKDhlKOdnZHK7JU="
```

Comment:
@micaswyers this is generated directly by the `jsdelivr` CDN. If you go t https://www.jsdelivr.com/package/npm/grapheme-splitter and click on "Copy HTML + SRI" you get the whole tag with the integrity sha. 

I'm planning to get rid of that in the HTML since the new `SegmentedMessage` class will expose directly the grapheme, so there is no need of explicit import in the HTML